sdk/attestation/azure-security-attestation/samples/policy/set_sealed_policy.cpp

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief This sample provides the code implementation to use the Key Vault Certificates SDK client
+ * for C++ to create, get, update, delete and purge a certificate.
+ *
+ * @remark The following environment variables must be set before running the sample.
+ * - ATTESTATION_ISOLATED_URL:  Points to an Attestation Service Instance in Isolated mode.
+ * - AZURE_TENANT_ID:     Tenant ID for the Azure account.
+ * - AZURE_CLIENT_ID:     The Client ID to authenticate the request.
+ * - AZURE_CLIENT_SECRET: The client secret.
+ * - ISOLATED_SIGNING_KEY: A Base64 encoded DER encoded RSA private key which matches the private
+ * key used when creating the ATTESTATION_ISOLATED_URL.
+ * - ISOLATED_SIGNING_CERTIFICATE: A Base64 encoded X.509 certificate wrapping the public key of the
+ * ISOLATED_SIGNING_KEY
+ *
+ */
+
+#include "get_env.hpp"
+
+#include "cryptohelpers.hpp"
+#include <azure/attestation.hpp>
+#include <azure/core/base64.hpp>
+#include <azure/core/cryptography/hash.hpp>
+#include <azure/core/internal/cryptography/sha_hash.hpp>
+#include <azure/identity.hpp>
+#include <cassert>
+#include <chrono>
+#include <iomanip>
+#include <iostream>
+#include <thread>
+#include <vector>
+
+// cspell:: words mrsigner mrenclave mitm
+using namespace Azure::Security::Attestation;
+using namespace Azure::Security::Attestation::Models;
+using namespace std::chrono_literals;
+using namespace Azure::Core;
+using namespace Azure::Core::Cryptography::_internal;
+
+int main()
+{
+  try
+  {
+    AttestationAdministrationClientOptions clientOptions;
+
+    // Attestation tokens returned by the service should be issued by the
+    // attestation service instance. Update the token validation logic to ensure that
+    // the right instance issued the token we received (this protects against a MITM responding
+    // with a token issued by a different attestation service instance).
+    std::string endpoint(std::getenv("ATTESTATION_ISOLATED_URL"));
```

Comment:
const.

I might be very annoying with all those `const` suggestions =/  
Think I'd just stop now.  It's not really relevant for samples

Reply:
I have updated every single variable in the samples to be const. Personally I disagree with adding "const" everywhere - to me, it is the same as hungarian and just adds friction in the future, but it's clearly very important to you, and it's not that big a deal to me.